### ISSUE TITLE
NVDA doesn't report items in Task Manager

### ISSUE THREAD

**Issue by Hatoon-fel** [nvaccess/nvda#14645]
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

### Steps to reproduce:
- open Task Manager with Ctrl+Shift+Escape or with any way you prefer.
- Press tab to move between items.
### Actual behavior:
NVDA reports Pane and Tab Control and doesn't read any item name when moving by Tab or arrows.
### Expected behavior:
NVDA should read items properly as previous versions.
#### NVDA installed/portable/running from source:
Portable
#### NVDA version:
NVDA2023.1 Beta
#### Windows version:
Windows 10 pro version 21H2 (19044.2204)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, I'm using installed nvda2022.4 and the items in task manager is reported as usual.
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi, portable NVDA does not run with admin privileges by default – you must be running installed copy of NVDA o interact with Task Manager. Thanks.

**Comment by Brian1Gaff** [comment]
So one would need to either reduce security, or somehow hack nvda portable 
to be seen as an admin task, which I'd suspect is very hard to do on modern 
windows.
 I'm not sure if its possible these days, but I tended to put the currently 
running installed version to a portable in a folder on the hard drive, then 
install the new one properly. You can then  see what happens, but if the new 
one is too buggy  or has other issues, do a complete uninstall, run your 
portable and re install the old one from the portable one, it normally then 
installs all the add ons as well, though some tweaking maybe needed 
particularly on synths and voices.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by seanbudd** [comment]
Closing as can't fix
